pyproject.toml

```diff
@@ -1,8 +1,39 @@
+[project]
+name = "pgocmmitfest"
+description = "Commitfest app for the PostgreSQL community"
+dynamic = ["version"]
+authors = [{ name = "xxx", email = "yyy" }]
+readme = "README.md"
+licsense = "PostgreSQL"
+dependencies = [
+    "django>=4.2,<5.0",
+    "psycopg2",
+    "simplejson",
+    "requests",
+    "pycryptodomex",
+]
+
+[project.optional-dependencies]
+dev = [
+    "uwsgi",
+    "pycodestyle",
+    "ruff",
+    "djhtml",
+]
+
+[tool.setuptools.packages.find]
+include = ["pgcommitfest*"]
+
+[tool.pycodestyle]
+statistics = true
+ignore = ["E402", "E501", "W503"]
+max-line-length = 120
+
 [tool.ruff]
-# line-length = 120
+line-length = 120
 
 [tool.ruff.format]
-# quote-style = "preserve"
+quote-style = "preserve"
```

Comment:
These were commented out on purpose. Let's just remove them, instead of enabling them.